Ada_Zangemann-cs.txt

```diff
@@ -5,55 +5,53 @@
 ## 0 ##
 
 [[IDX:
-toc:Table of Contents
+toc:Obsah
 text:Text
-cover:Cover
-other.titlepage:Front Cover
-chapter-01,cover:Front Cover
+cover:Přebal
+other.titlepage:Titulní strana
+chapter-01,cover:Titulní strana
 chapter-02,page_02:Imprint
 chapter-03,page_04:Ada & Zangemann
-chapter-04,page_06:Zangemann and computers
-chapter-05,page_13:One day...
-chapter-06,page_18:Skateboards no longer work
-chapter-07,page_20:Ada and software
-chapter-08,page_22:Programming
-chapter-09,page_25:A horrible night
-chapter-10,page_26:First real project
-chapter-11,page_30:Hacking for freedom
-chapter-12,page_36:Zangemann and the President
+chapter-04,page_06:Zangemann a počítače
+chapter-05,page_13:Jednoho dne...
+chapter-06,page_18:Skateboardy přestaly fungovat
+chapter-07,page_20:Ada a software
+chapter-08,page_22:Programování
+chapter-09,page_25:Hrozná noc
+chapter-10,page_26:První skutečný projekt
+chapter-11,page_30:Hackování za svobodu
```

Comment:
"hackovaní" se dětem bude blbě číst. Ale nenapadá mě nic lepšího :)

Reply:
Souhlasím. Taky mě nenapadala alternativa. 
Jedině buďto úplně uhnout od originálu a přidat nějakou "editor's note" s odkazem na originální název a vysvětlením "hackingu".
Nebo přes asterisk odkázat na fonetickou variantu. To mi přijde ale dost rušivé.